set-web/src/main/java/eu/europeana/set/web/service/impl/UserSetServiceImpl.java

```diff
@@ -493,6 +493,27 @@ public UserSet fetchDynamicSetItems(UserSet userSet, String apiKey, String sort,
 	}
     }
 
+    @Override
+    public UserSet fetchDynamicSetItemDescriptions(UserSet userSet, String apiKey, String sort, String sortOrder, int pageNr,
+	    int pageSize) throws HttpException {
+
+	String url = buildSearchApiUrl(userSet, apiKey, sort, sortOrder, pageNr, pageSize);
+
+	SearchApiResponse apiResult;
+	try {
+	    apiResult = getSearchApiClient().searchItemDescriptions(url, apiKey, "UserSet.FETCH_ITEMS");
+	    setItemDescriptions(userSet, apiResult);
+	    return userSet;
+	} catch (SearchApiClientException e) {
+	    if (SearchApiClientException.MESSAGE_INVALID_ISSHOWNBY.equals(e.getMessage())) {
+		throw new RequestBodyValidationException(I18nConstants.USERSET_VALIDATION_PROPERTY_VALUE,
+			new String[] { WebUserSetFields.IS_DEFINED_BY, userSet.getIsDefinedBy() });
+	    } else {
+		throw new InternalServerException(e);
+	    }
+	}
+    }
+
     private void setItems(UserSet userSet, SearchApiResponse apiResult) {
```

Comment:
Setting item descriptions must probably not be set in the items, because of serialization reasons. 
the item descriptions must not be json escaped in the serialization, the output must be proper json

Reply:
When item descriptions must not be set in the items, then where?